File: PeopleManagementApi/Models/People.cs
namespace PeopleManagementApi.Models
{
    public class People
    {
        public long Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Phone { get; set; }
        public string Location { get; set; }
        public string Email { get; set; }
        public int YearsOfExperience { get; set; }
        public Job Parent { get; set; }
        public ICollection<Skill> Skills { get; set; }
    }
}

File: PeopleManagementUi/src/app/models/people.ts
import { Skill } from "./skill";

export interface People
    {   
        id?: number;
        firstName?: string;
        lastName?: string;
        phone?: string;
        location?: string;
        email?: string;
        yearsOfExperience?: number;
        skills?: Skill[];

        preparingDelete?: boolean;
        isEditing?: boolean;
    }